getsolus/packages
# mlt: Update to 7.24.0

**Summary**

- Full release note can be found [here](https://github.com/mltframework/mlt/releases/tag/v7.24.0)
- Enable "Motion Tracker" effect in `kdenlive` (Resolves getsolus/packages#2465)
- Add monitoring.yml

**Test Plan**


- Launch `kdenlive`
- Make sure "Motion Tracker" effect is available
- Launch `shotcut`

**Checklist**
